### ISSUE TITLE
Compile plugin on Android

### ISSUE THREAD

**Issue by i02bahip** [mrayy/mrayGStreamerUnity#14]
Hi!
First, thanks for this amazing plugin!. Once I have managed to compile the plugin for windows, now I want to compile it for android, but I can't see clear instructions to do that. I can see in Android folder a build.bat and cleanup.bat. The first generate the .so file and the second move it to the "correct" folder. I'm able to do that, but I think to do something more that I don't know, b'cos when I .build and run in my phone the first test scene, the videotest texture doesn't appear. So, any update in "how to compile on Android" steps would be appreciated for the unexperienced people in unity.. like me :) 

Thanks!

**Comment by mrayy** [comment]
The Android branch I was working on it like 7months ago, and I had it running back then. I forgot to document it back then, but in the git repo, I have the pre-built .so libraries included. Could you try to use the already existing ones and build them for android and test?

**Comment by i02bahip** [comment]
Hi Again!

I've tried to compile in the "android" branch with ndk r17c. I have assumed the following:
- For compile the android libraries, I need to launch the "build.bat", and then "cleanup.bat".
- I suppose the main target is to create a .so file and replace it in some place.

The first doubt, is I don't know even if that assumptions are right or not.
The second doubt, is what "build.bat" and "cleanup.bat" I need to launch, b'cos I see them in Android folder, and in Android/Gstreamer folder too. I can generate "libgstreamer_android.so" with both build.bat but with none of them seems that the plugin works when I replace this .so in "..mrayGStreamerUnity\Unity\UnityTests\Assets\Plugins\Android" (and actually, I don't know if is the right place)  I build and run an android application for the first test scenario, and I only see a solid purple color as texture. 

In summary... I have a lot of doubts! haha. So this is why I would appreciated so much a little help, like the right steps to follow to compile the library in android.

Thanks again!! Regards. (and sorry for my cheap english! haha)

**Comment by keli95566** [comment]
Hi there! @i02bahip were you able to build on Android? I ran into the same problem here and could very gratefully use some help, especially for building arm64 and opengles3!! Thank you so much!!
